stores/amazon.py

```diff
@@ -179,10 +184,18 @@ def something_in_stock(self):
         f.set(params)
         self.driver.get(f.url)
         self.check_if_captcha(self.wait_for_pages, ADD_TO_CART_TITLES)
-        if self.driver.find_elements_by_xpath('//td[@class="price item-row"]'):
-            log.info("One or more items in stock!")
+        price_element = self.driver.find_elements_by_xpath('//td[@class="price item-row"]')
+        if price_element:
+            price = float(price_element[0].text[1:])
```

Comment:
Sorry, forgot to include this before. This fails if the price has a comma in it. Could use package that has locale, which would be more robust, but for US (or anywhere that uses comma for separating digit groupings):
```suggestion

            price_str = price_element[0].text[1:]
            price = float(str_price.replace(',',''))
```